README.md

```diff
@@ -20,7 +20,34 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+Geocoding and reverse geocoding will work in the same way as provided by `geocoder` gem.
+
+To use autocomplete API
+
+```ruby
+Geocoder.search('Delhi', lookup: :here_suggest)
+```
+
+To use calculateroute API
+
+```ruby
+Geocoder.search([[26.1215649, -80.1273346], [34.8440896, -82.4057]],
+                lookup: :here_calculate_route,
+                params: { mode: 'fastest;truck', departure: "now", routeattributes: 'shape'}).first
+```
+
+
+If multiple lookups are used in application no need to define separate configuration for these new lookups in `geocoder.rb`. They will use the configuration defined for here maps.
+
+```ruby
+here: {
+  api_key: [YOUR_APP_ID, YOUR_APP_CODE],
+  cit: true/false
+}
+
+```
+
+`cit` options allows for usage of customer integration testing urls defined by HERE maps documentation.
```

Comment:
This can be more appropriate - 

```
`cit` configuration option enables usage of integration testing URLs mentioned in HERE maps documentation`
```

Reply:
@shailesh-kalamkar  removed this altogether since this option is no longer available.